arbor/server/core/config.py

```
@@ -21,17 +21,62 @@ class ArborConfig(BaseModel):
 
 class Settings(BaseModel):
 
-    STORAGE_PATH: str = "./storage"
+    # STORAGE_PATH: str = "./storage"
+    STORAGE_PATH: str = str(Path.home() / ".arbor" / "storage")
     INACTIVITY_TIMEOUT: int = 30  # 5 seconds
     arbor_config: ArborConfig
+    
+    def __init__(self, **data):
+        super().__init__(**data)
+        
+        # Create ~/.arbor directories if it DNE
+        self._init_arbor_directories()
+
+
+    def _init_arbor_directories(self):
+        arbor_root = Path.home() / ".arbor"
+        config_dir = arbor_root / "config"
+        storage_dir = Path(self.STORAGE_PATH)
+        
+        arbor_root.mkdir(exist_ok=True)
+        config_dir.mkdir(exist_ok=True)
+        storage_dir.mkdir(exist_ok=True)
+        (storage_dir / "logs").mkdir(exist_ok=True)
+        (storage_dir / "models").mkdir(exist_ok=True)
+        (storage_dir / "uploads").mkdir(exist_ok=True)
+
+    @classmethod
+    def find_config_path(cls) -> Optional[str]:
```

```
    @classmethod
    def find_config_path(cls) -> Optional[str]:
        """ Search 1st for: ~/.arbor/config/default.yaml 2nd: ./arbor.yaml (backward compatibility)"""

        # Check ~/.arbor/config
        arbor_config = Path.home() / ".arbor" / "config.yaml"
        if arbor_config.exists(): 
            return str(arbor_config)
        
        return None
```

I think just config.yaml is fine. I don't think there is a huge use case for different config paths within a config directory. Also no need for a fallback here, if its not obvious which one we should be using, we can just throw an error and tell the user to use the cli to setup a config